include/libtorrent/torrent.hpp

```diff
@@ -1287,7 +1285,7 @@ namespace libtorrent
 		// the posix time this torrent was added and when
 		// it was completed. If the torrent isn't yet
 		// completed, m_completed_time is 0
-		time_t m_added_time;
+		time_t m_added_time = time(nullptr);
```

Comment:
this was not a semantic change. The initialization was just moved from torrent.cpp:183